pxtpy/converter.ts

```diff
@@ -1391,7 +1391,10 @@ namespace pxt.py {
         Break: (n: py.Break) => B.mkStmt(B.mkText("break")),
         Continue: (n: py.Continue) => B.mkStmt(B.mkText("break")),
 
-        Delete: (n: py.Delete) => stmtTODO(n),
+        Delete: (n: py.Delete) => {
+            error(n, 9550, U.lf("delete statements are unsupported"));
```

Comment:
I think delete might work now / there might be support for it? not related to this PR really, will take a look later